src-test/src/org/openbravo/financial/FinancialUtilsTest.java

```diff
@@ -0,0 +1,35 @@
+package org.openbravo.financial;
+
+import java.util.Date;
+
+import org.openbravo.model.common.businesspartner.BusinessPartner;
+import org.junit.Test;
+import org.openbravo.base.exception.OBException;
+import org.openbravo.dal.service.OBDal;
+import org.openbravo.base.weld.test.WeldBaseTest;
+import org.openbravo.model.pricing.pricelist.PriceList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+
+public class FinancialUtilsTest extends WeldBaseTest {
+  protected static final String BP_HEALTHY_FOOD = "B3ABB0B4AFEA4541AC1E29891D496079";
+
+
+  @Test
+  public void testGetProductPriceWithNullProduct() {
+    // Given
+    Date date = new Date();
+    boolean useSalesPriceList = true;
```

Comment:
This variable is just used once, you can use "true" on the parameter for the method getProductPrice(...)